RELEASING.md

```diff
@@ -2,6 +2,7 @@
 
 This page describes how to cut releases of our plugin. It's strictly for the owners of the project.
 
+1. Update [CHANGELOG.md](https://github.com/GoogleCloudPlatform/google-cloud-intellij/blob/master/CHANGELOG.md) with any major release notes (see [guidelines](http://keepachangelog.com/en/0.3.0/)). Replace the contents of the `<change-notes>` tag in [plugin.xml](https://github.com/GoogleCloudPlatform/google-cloud-intellij/blob/master/google-cloud-tools-plugin/resources/META-INF/plugin.xml) with the same notes, in html.
```

Comment:
Maybe a link to a markdown compiler? If you think that's inappropriate maybe change "in html" to .."with the same notes compiled from markdown to html".